_projects/de/2020-03-06-TTN-Mapper.md

```diff
@@ -27,6 +17,19 @@ addons: ["LoRa-Bee"]
 lang: de
 tags: ["Geographie", "Informatik", "LoRa", "TTN", "Blockly"]
```

Comment:
@PaulReichmuth could you also add the tag `LoRaWAN`

Reply:
One sec, will do

Reply:
@mpfeil done, should be in now